Index: src/app/album/album-detail/album-detail.component.ts
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/app/album/album-detail/album-detail.component.ts	(date 1570421952296)
+++ src/app/album/album-detail/album-detail.component.ts	(date 1570421952296)
@@ -19,7 +19,6 @@
   private songList: any[];
   private playList: Playlist;
   private subscription: Subscription;
-  i: number;
 
   constructor(
     private playlistService: PlaylistService,
